### PR TITLE
Edit-UG

### PR DESCRIPTION
**Pull request by MagnificentCreature**
Add sections on:
- If you're new: What are commands
- Command macros

Fix documentation bugs/Refactored:
- Long note on the command format
- Profile uniqueness
- Positioning of commands in Viewing section
- Interaction
- Added more table of contents
- Added dropdown for technical explanation of reminders

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -177,9 +193,13 @@ t/friends tg/@yuyubern pf/Graphic Designer i/60000
 
 You should directly see the client profile added to the list of clients in the application window. After adding a client profile, you can now perform various operations on the client profile as specified in the next few sections!
 
+[↑ Back to Table of Contents](#table-of-contents)
+
 ### Adding a client interaction: ***interaction***
 
-After adding a client profile, you can now log your interactions with the client.  Use the ***interaction*** command to add an interaction to a client profile.
+After adding a client profile, their information is now recorded and easily accessible!
+But you may also want a way to log your interactions with the client.
```
Comment (KamiliArsyad) [inline docs/UserGuide.md]: "But" shouldn't be at the start of a sentence and it's not really negating the statement above it no? Maybe can rephrase this part

File: docs/UserGuide.md [hunk 2]
```diff
@@ -321,47 +330,72 @@ find David
...
+| COLD     | 8 weeks        |
+| UNKNOWN  | 4 weeks        |
+
+
+Say your last interaction with John is on 1st Jan 2021 and he is a HOT lead. The follow-up date will be 1 week after 1st Jan 2021, which is 8th Jan 2021. 
+</details>
+
+[↑ Back to Table of Contents](#table-of-contents)
+
+### Viewing the list of clients: ***list***
```
Comment (KamiliArsyad) [inline docs/UserGuide.md]: Would it be better to have list of clients above as before? We can only see the clients after going from dashboard to list

**Review by KamiliArsyad — APPROVED** [review]
oki

File: docs/UserGuide.md (hunk 1 above)
Comment (MagnificentCreature) [reply docs/UserGuide.md]: fixed!

File: docs/UserGuide.md (hunk 2 above)
Comment (MagnificentCreature) [reply docs/UserGuide.md]: Okay true...

**Review by dickongwd — APPROVED** [review]
LGTM

**Pull request merged**